docs/src/examples/BayesianNODE_NUTS.md

```diff
@@ -0,0 +1,278 @@
+# Bayesian Neural ODEs: NUTS
```

Comment:
why only NUTS?

Reply:
In this tutorial, only NUTS is shown. 

For SGLD, we can put it in another tutorial. Else, I feel this tutorial will be too big.